docs/release-notes/release-notes-0.19.0.md

```diff
@@ -103,6 +103,10 @@
 * The `lncli wallet fundpsbt` command now has a [`--max_fee_ratio` argument to
   specify the max fees to output amounts ratio.](https://github.com/lightningnetwork/lnd/pull/8600)
 
+* The `lncli addinvoice --blind` command now has the option to include a chained
+  channels incoming list [`--blinded_path_incoming_channel_list]
```

Comment:
consider adding more context to the description so that the objective of the change is clear to the user...
e.g.
...channels incoming list [`--blinded_path_incoming_channel_list] which gives users the control of specifying the channels they prefer to receive the payment on. With the option to specify multiple channels this control can be extended to multiple hops leading to the node.